src/applications/qlippy-frontend/app/globals.css

```
@@ -28,6 +92,321 @@ body {
+        animation: fadeIn var(--transition-normal) forwards;
+    }
+    
+    .animate-slide-up {
+        animation: slideUp var(--transition-normal) forwards;
+    }
+    
+    .animate-glow {
+        animation: glow 2s ease-in-out infinite alternate;
+    }
+    
+    @keyframes animateIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "animateIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)